### ISSUE TITLE
There are some problems in program testing, I hope you have a look.

### ISSUE THREAD

**Issue by jia-github-pro** [yunshengtian/AutoOED#3]
Your project is very valuable. When I experience the relevant function, run_team_scientist.py, the basic method described in the documentation is that the whole program stops responding after the manual optimization is started and before the optimization is finished. I hope you can check it out.

**Comment by yunshengtian** [comment]
Thanks for your feedback! I'll look into this issue soon.

Just to double check if my understanding is correct, does this issue happen when you click the manual optimization button while following the basic steps in the documentation?

**Comment by jia-github-pro** [comment]
Yes, I use the team version. It is also possible that I installed a version of the library that is slightly higher than the version you provided, such as Pymoo. However, run_personal.py doesn't have the stopping responding problem,  so I don't think it's the version of the installed library. The example I used is: https://autooed.readthedocs.io/en/latest/content/getting-started/basic-usage.html
